loadgen/loadgen.h

```diff
@@ -28,6 +29,8 @@ class SystemUnderTest;
 struct TestSettings;
 struct LogSettings;
 
+using EndOnDeviceCallback = std::function<void(QuerySampleResponse*)>;
```

Comment:
Can we make this function type name more generic? For example, ResponseCallback or something like that. This is because in the future there may be some untimed post-processing that can make use of this callback feature.

Reply:
Sure, good point